lib/document/templates.js

```diff
@@ -111,6 +163,7 @@ ${documentStyleContent}
 ${titlePageStyle(node)}
 ${customStyleContent(node)}
 </style>
+${mathjaxContent(node)}
```

Comment:
The following line will become `${mathjax.content(node)}`.

Reply:
changed.